docs/tutorials/tutorial-0-Linux.rst

```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+===============================================
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a working Ubuntu Linux project.
```

Comment:
Again - there's nothing Ubuntu specific here. The text should refer to this being a "Linux" tutorial, with a note somewhere near the top (around this spot would be fine) that reads something like this:

```
.. note:
    This tutorial has been tested with Ubuntu 16.04; however it should work on any Linux.
```